packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -513,4 +513,54 @@ export function AriaLiveRegion(regionName, elem, assertive = false) {
     dispatchMessage,
     registerRegion,
   };
+}
+
+/**
+ * Checks whether an element is focusable.
+ * @param {HTMLElement | Element}  element
+ * @returns {boolean}
+ */
+export function isFocusable(element) {
+  // Source: https://stackoverflow.com/a/30753870
+  if (element.getAttribute('tabindex') === '-1') {
+    return false;
+  }
+  if (element.hasAttribute('tabindex') || element.getAttribute('contentEditable') === 'true') {
+    return true;
+  }
+  switch (element.tagName) {
+    case 'A':
+    case 'AREA':
+      return element.hasAttribute('href');
+    case 'INPUT':
+    case 'SELECT':
+    case 'TEXTAREA':
+    case 'BUTTON':
+      return !element.hasAttribute('disabled');
+    case 'IFRAME':
+      return true;
+    default:
+      return false;
+  }
+}
+
+/**
+ * Returns the last focusable element of for an HTML element. 
+ * This function would NOT work with shadow root.
+ * @param {Element | null} element
+ * @returns {Element | null}
+ */
+ export function getLastFocusableElement(element) {
+  if (!element || element.nodeType === 3) return null;
```

Comment:
Please use the constant instead. See https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType#value

```suggestion
  if (!element || element.nodeType === Node.TEXT_NODE) return null;
```